plugins/inputs/sqlserver/azuresqldbqueries.go

```diff
@@ -233,13 +233,27 @@ SELECT
 		WHEN slo.[edition] = 'Hyperscale' then NULL
 		ELSE CAST(DATABASEPROPERTYEX(DB_NAME(),'MaxSizeInBytes') as bigint)/(1024*1024)
 	END AS [total_storage_mb]
+	,CASE
+		WHEN slo.[edition] = 'Hyperscale' then NULL
```

Comment:
This special case for Hyperscale should not be used here as well as in the definitions of `total_log_mb` and `used_space_log_mb` columns. These values are available for Hyperscale in the same way as for other service tiers.

The special case is only needed in the definition of `available_storage_mb` because for Hyperscale, the `MaxSizeInBytes` property is not defined (it returns -1). In that case, returning NULL does make sense because if maximum size is not defined, then available storage is not defined either.